app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationViewModel.kt

```diff
@@ -141,7 +174,7 @@ class ConversationViewModel(
     val isMessageRequestThread : Boolean
         get() {
             val recipient = recipient ?: return false
-            return !recipient.isLocalNumber && !recipient.isGroupRecipient && !recipient.isApproved
+            return !recipient.isLocalNumber && !recipient.isLegacyGroupRecipient && !recipient.isCommunityRecipient && !recipient.isApproved
```

Comment:
You are only checking for legacy here. Are groupsV2 not part of this?

Reply:
So groupsv2 **can** have message request thread hence not appearing in the `!` check